docs/setting_up_VM_with_app.md

```diff
@@ -0,0 +1,27 @@
+## Steps to follow to get a VM with monitoring app running on Apache
+
+To get a 'prototype' of the monitoring app running with Apache, follow these steps:
+- create a cloud VM of the type: scientific-linux-7-aq
+- continue by selecting sandbox 'testing_personality_2', archetype 'ral-tier1', personality 'apel-data-validation-test'
+
+Allow 15 minutes after the machine is created, then remember to edit security groups from OpenStack to allow Apache to work.
+Then follow these steps from within the machine:
+- yum remove git && quattor-fetch && quattor-configure --all
```

Comment:
```suggestion
- quattor-fetch && quattor-configure --all
```

I assume this was meant to say "and install git236", don't think this is needed anymore.